apps/spruce/src/analytics/waterfall/useWaterfallAnalytics.ts

```diff
@@ -23,7 +23,8 @@ type Action =
   | { name: "Changed page"; direction: "next" | "previous" }
   | { name: "Deleted one filter chip" }
   | { name: "Deleted all filter chips" }
-  | { name: "Toggled task icon legend"; open: boolean };
+  | { name: "Toggled task icon legend"; open: boolean }
+  | { name: "Viewed waterfall beta modal"; enabledBeta: boolean };
```

Comment:
nit. I think the attribute doesn't follow our typical conventions (different works should be separated by dots). Maybe we can create a namespace that will be reusable for other things

Reply:
Changed to `beta.enabled`